common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsAuthorizationResultFactory.java

```diff
@@ -60,14 +60,24 @@ public MicrosoftStsAuthorizationResult createAuthorizationResult(final int resul
         MicrosoftStsAuthorizationResult result;
         switch (resultCode) {
             case AuthenticationConstants.UIResponse.BROWSER_CODE_CANCEL:
-                Logger.verbose(TAG, null, "User cancel the request in webview.");
+                Logger.verbose(TAG, null, "User cancel the authorization request in UI.");
                 result = createAuthorizationResultWithErrorResponse(AuthorizationStatus.USER_CANCEL,
                         MicrosoftAuthorizationErrorResponse.USER_CANCEL, MicrosoftAuthorizationErrorResponse.USER_CANCELLED_FLOW);
                 break;
 
             case AuthenticationConstants.UIResponse.BROWSER_CODE_COMPLETE:
-                final String url = data.getStringExtra(MSSTS_AUTHORIZATION_FINAL_URL);
-                result = parseUrlAndCreateAuthorizationResponse(url, data.getStringExtra(MicrosoftAuthorizationResult.REQUEST_STATE_PARAMETER));
+
```

Comment:
I'm not seeing how this change differs from the previous behavior.... I can see that we check for a `null` URL inside of `parseUrlAndCreateAuthorizationResponse`; is this check necessary here?